### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR adds issue templates.

I took the issue templates from the https://github.com/corona-warn-app/cwa-app-ios repository and adapted them to this project. I asked for permission to use these templates in https://github.com/corona-warn-app/cwa-documentation/issues/652 and the feedback was that this is ok. 

If you have any change requests, please, let me know. 
If you like these issue templates, it would be no problem for me to also open a PR in the other repositories here. 

Thank you for your great work!

### CODE REVIEW

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
Indeed, these templates are used on the CWA project to streamline the process at the interface between the  community and developers. At CWA the information is then transferred to internal JIRA tickets.

---
Corona-Warn-App Open Source Team

**Comment by timokoenig** [comment]
I like the issue templates and I think it's a good idea to have it similar to the CWA project. Thanks for that!

I will check with the internal team and come back to it

File: .github/ISSUE_TEMPLATE/01_bugs.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+-->
+
+## Avoid duplicates
+* [ ] Bug is not mentioned in the [FAQ](https://digitaler-impfnachweis-app.de/faq/)
+* [ ] Bug is not already reported in another issue
+
+## Technical details
+
+- Device name:
+- iOS version:
```
Comment (timokoenig) [inline .github/ISSUE_TEMPLATE/01_bugs.md:22]: Maybe we should use `OS version` so we can use the exact template for Android

Comment (Ein-Tim) [reply .github/ISSUE_TEMPLATE/01_bugs.md:22]: Will change ASAP. Should I prepare a PR for the Android repo?

Comment (Ein-Tim) [reply .github/ISSUE_TEMPLATE/01_bugs.md]: Done in https://github.com/Digitaler-Impfnachweis/covpass-ios/pull/27/commits/2150c425936c7b8a8844854252438011e3273bc0.

**Pull request merged**